packages/manager/apps/hub/src/index.scss

```diff
@@ -6,6 +6,20 @@
 @import '@ovh-ux/manager-hub/src/variables.scss';
 @import './assets/theme/index.scss';
 
+.hub-main-view {
+  > * {
+    max-width: $hub-view-max-width;
+  }
+}
+
+.overflow-hidden {
```

Comment:
Since version 4.2 these helpers are available.

Maybe we can consider a bump to the version 4.4.

Reply:
You're right ! this is done :)